File: api/model/Sheet.py
class Sheet:
    def __init__(self):
        self.answers = []
        self.question_count = 0
        self.empty_count = 0
        self.multi_answer_count = 0
        self.options = 0
        self.image_path = ""

    def __str__(self):
        return f"Question Count: {self.question_count}, Empty Count: {self.empty_count}, Multi Answer Count: {self.multi_answer_count}, Options: {self.options}, Image Path: {self.image_path}, Answers: {self.answers}"

File: lib/models/sheet.dart

class Sheet {

  int question_count;
  int options;
  List<String> answers;

  Sheet(this.question_count, this.options, this.answers);

  @override
  String toString() {
    return 'Sheet{soru_sayisi: $question_count, sik_sayisi: $options, cevaplar: $answers}';
  }


  static Sheet fromJson(Map parsed) {
    return Sheet(parsed['question_count'], parsed['options'], List<String>.from(parsed['answers']));
  }
}